coderd/coderdtest/coderdtest.go

```diff
@@ -111,6 +111,8 @@ type Options struct {
 	// test instances are running against the same database.
 	Database database.Store
 	Pubsub   database.Pubsub
+
+	SwaggerEndpointEnabled bool
```

Comment:
this should be in the first block with the other options that get passed through to coderd.Options

Reply:
Fixed